record-and-playback/core/scripts/process/captions.rb

```diff
@@ -0,0 +1,106 @@
+# Set encoding to utf-8

```

Comment:
This script isn't a recording processing format, so it can not be in the `process` directory. For the lack of a better place to put it, the `utils` subdirectory is ok.